js/index.js

```diff
@@ -1,20 +1,211 @@
 const currentDate = new Date();
-const thisYear = currentDate.getFullYear(); 
+const thisYear = currentDate.getFullYear();
+
+const footer = document.createElement("footer");
+const copyright = document.createElement("p");
+copyright.innerHTML = `<small>Helen Abera &copy; ${thisYear}</small>`;
+
+footer.appendChild(copyright);
+document.body.appendChild(footer);
+
+let skills = ["Project Management", "Health Care Admin", "JavaScript", "CSS", "HTML"];
+let skillsSection = document.getElementById("mySkills");
+let skillsList = document.createElement("ul");
+skillsSection.appendChild(skillsList);
+
+for (let skill of skills) {
+    const skillItem = document.createElement("li");
+    skillItem.innerText = skill;
+    skillsList.appendChild(skillItem);
+}; 
+
+
+let messageForm = document.querySelector("[name='leave_message]"); 
+let messageSection = document.getElementById("messages"); 
+let messageList = messageSection.querySelector("ul")
+messageSection.hidden = true; 
+
+let idCounter = 0;
+
+function makeId() {
+    let id = "entry" + idCounter++;
+    return id; 
+}
+
+let entryById={}; 
+
+messageForm.addEventListener("submit", (event) => {
+    event.preventDefault();
+    let name = event.target.usersName.value; 
+    let email = event.target.usersEmail.value; 
+    let message = event.target.usersMessage.value; 
+
+    console.log("Name:", usersName);
+    console.log("Email:", usersEmail);
+    console.log("Message:", usersMessage);
+    let uid = makeId();
+    let newMessage = document.createElement("li");
+    newMessage.classList.add("message-item"); 
+
+    newMessage.innerHTML = `<a href="mailto:${email} ">${name}  </a><span>wrote: ${message}</span> `
+    newMessage.setAttribute("id", uid); 
+
+    entryById[uid] = {usersName: name, usersEmail: email, usersMessage: message}; 
+    newMessage.appendChild(makeEditButton());
+    newMessage.appendChild(makeRemoveButton()); 
+
+    messageList.appendChild(newMessage);
+    messageForm.requestFullscreen();
+    messageSection.hidden = false; 
+
+
+}); 
+/*
```

Comment:
Use the versioning system (git) to keep track of old versions rather than keeping commented out code around.